### PR TITLE
Various tailwind fixes

### PR DESCRIPTION
**Pull request by robertmcabee**
This is intended to improve the default `index.tsx` page.

- Removes NextJS `<Link>` component
- Whole card is clickable
- Hover on card changes button styles
- Hover animation duration reduced from 500ms to 300ms
- changed parent `<section>` to `<main>` for better semantics
- Card heights are consistent on each row

This is my first contribution to the project, hopefully everything looks okay!

### CODE REVIEW

File: template/page-studs/index/with-auth-trpc-tw.tsx [hunk 1]
```diff
@@ -42,62 +41,74 @@ const Home: NextPage = () => {
...
-            <p className="mb-5">
-              Strongly typed programming language that builds on JavaScript,
-              giving you better tooling at any scale
-            </p>
-            <button className="p-2 px-6 text-white bg-blue-500 rounded-md hover:bg-blue-600">
-              <Link href="https://www.typescriptlang.org/">
-                <a target="_blank">Learn More</a>
-              </Link>
-            </button>
+          <section className="h-full transform group border-2 border-neutral-800 rounded-2xl duration-300 hover:scale-105 hover:border-blue-600 hover:-translate-y-1 hover:shadow-2xl">
```
Comment (TheoBr) [inline template/page-studs/index/with-auth-trpc-tw.tsx:58]: Is `hover:-translate-y` correct? The extra - at the beginning in particular

**Review by TheoBr — APPROVED** [review]

File: template/page-studs/index/with-auth-trpc-tw.tsx (hunk 1 above)
Comment (robertmcabee) [reply template/page-studs/index/with-auth-trpc-tw.tsx:58]: I believe so - its just a negative value in tailwind, i.e. negative margin on hover would be `hover:-mt-8`. It's from the initial tailwind styles, not my code so I'm not 100% confident.

**Comment by nexxeln** [comment]
We have some merge conflicts. Take a look at #89

**Comment by robertmcabee** [comment]
Closing per nexxel to hopeful making merging easier. Will submit a new PR.

**Pull request closed without merging**